src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -85,7 +85,7 @@ export class CellModel extends Disposable implements ICellModel {
 	private _outputCounter = 0; // When re-executing the same cell, ensure that we apply chart options in the same order
 	private _attachments: nb.ICellAttachments | undefined;
 	private _preventNextChartCache: boolean = false;
-	private _lastEditMode: string | undefined;
+	public lastEditMode: string | undefined;
```

Comment:
Instead of exposing this why not test the functionality that uses it? i.e. change modes and then toggle isEditMode and  check that showPreview and showMarkdown have the expected values.

This doesn't really make sense to expose publicaly (especially on the interface) and testing the more broad scenario would be better from a testing standpoint anyways since that'd be testing what the purpose of this property is actually for.